mocha-casperjs.js

```diff
@@ -48,7 +48,6 @@ module.exports = function (Mocha, casper, utils) {
   })
 
   casper.on('waitFor.timeout', function(timeout, details) {
-    casper.step++
```

Comment:
I deleted it in the `step.timeout` handler. If I delete it in the `timeout` handler, some tests will fail.